python/sparkdl/transformers/param.py

```diff
@@ -122,3 +134,49 @@ def converter(value):
                 return value
             else:
                 raise TypeError("%s %s is not in the supported list." % type(value), str(value))
+        return converter
+
+
+class HasOutputTensor(Params):
+    outputTensor = Param(Params._dummy(), "outputTensor",
+                         "Name of output tensor in signature def",
+                         typeConverter=SparkDLTypeConverters.toStringOrTFTensor)
+
+    def __init__(self):
+        super(HasOutputTensor, self).__init__()
+
+    def setOutputTensor(self, value):
+        return self._set(outputTensor=value)
+
+    def getOutputTensor(self):
+        return self.getOrDefault(self.outputTensor)
+
+
+class HasInputTensor(Params):
```

Comment:
Normally, we need to handle multiple 1+ input tensors. See TensorFlow Serving as a reference.